mkosi/__init__.py

```diff
@@ -4842,6 +4843,13 @@ def create_parser() -> ArgumentParserMkosi:
         metavar="PATH",
         help="Use the specified private key when using `mkosi ssh` (requires a corresponding public key)",
     )
+    group.add_argument(
+        "--ssh-timeout",
+        metavar="SECONDS",
+        type=int,
+        default=0,
+        help="Wait up to SECONDS seconds for the SSH connection to be available when using `mkosi ssh`",
```

Comment:
This should also be added to the docs. 

Should we also document, that this is mainly to wait for the devices to come online, since there is the 20s loop to ping neighbours?

Reply:
I knew I forgot something, thanks. I added a documentation snipped mentioning the use case.